README.md

```diff
@@ -126,6 +126,8 @@ The following environment variables can be populated to run commands whenever sp
 | `POST_RESTART_HOOK` |  | Command to be executed after a server restart was performed. Future restarts and update checks are blocked until this command returns. |
 | `PRE_SERVER_RUN_HOOK` |  | Command to be executed before the server is started. Server startup is blocked until this command returns. |
 | `POST_SERVER_RUN_HOOK` |  | Command to be executed after the server has finished running. Server shutdown is blocked until this command returns or a shutdown timeout is triggered after 29 seconds. |
+| `PRE_SERVER_SHUTDOWN_HOOK` |  | Command to be executed before the server is shut down. Server shutdown is blocked until this command returns. |
```

Comment:
We should also make a note that after 90 seconds the entire process will be hard killed by supervisord.
See https://github.com/lloesche/valheim-server-docker/blob/286945daa4f4c98608f9f5357d8792d8be4eaf38/supervisord.conf#L72